source/agora/node/FullNode.d

```diff
@@ -877,6 +878,20 @@ public class FullNode : API
             this.config.validator, this.params);
     }
 
+    /***************************************************************************
+
+        Returns an instance of a DNSResolver
+
+        Returns:
+            the dns resolver
+
+    ***************************************************************************/
+
+    protected DNSResolver makeDNSResolver ()
```

Comment:
Well technically it's not instantiated (looking at the commit subject) because this is never called.